mastracode/factory-ui/src/ui/domains/factory/metrics-theme.css

```
@@ -0,0 +1,44 @@
+/* Color ramps for the Metrics page charts (QueueHealthChart, ShareBar).
+ * Imported from tailwind.css, not from the components: `@theme` only produces
+ * utilities inside the Tailwind entry's import graph. */
+
+/* sequential, not categorical — lightness carries the escalation, hue drifts sand → red */
+:root {
+  --queue-fresh: oklch(0.55 0.055 70);
+  --queue-aging: oklch(0.66 0.105 62);
+  --queue-stale: oklch(0.75 0.145 52);
+  --queue-critical: oklch(0.71 0.18 32);
+
+  --chart-soft-1: oklch(0.62 0.19 285);
```

Stylelint (17.14.1)

[error] Expected no empty line before custom property (custom-property-empty-line-before)

(custom-property-empty-line-before)